share/translations/zh-cn.txt

```diff
@@ -0,0 +1,81 @@
+113:晴                                    : Clear
+113:晴                                    : Sunny
+116:少云                                  : Partly cloudy
+119:多云                                  : Cloudy
+122:阴                                    : Overcast
+143:轻雾                                  : Mist
+176:可能下小雨                             : Patchy rain possible
+179:可能下小雪                             : Patchy snow possible
+182:可能有雨夹雪                            : Patchy sleet possible
+185:可能有冻毛毛雨                          : Patchy freezing drizzle possible
+200:可能打雷                               : Thundery outbreaks possible
```

Comment:
I saw the Russian translation used `местами`, which meant "局部 somewhere" rather than "可能 maybe". Should we change these translations?
- 局部有雨
- 局部有雪
- 局部雨夹雪
- 局部冻雨
- 局部雷雨

Reply:
@liyishuai
Russian word "местами" can be translated rather as "patchy" and not as "possible",
so you suggest using `局部`? Should I just replace it everywhere? On only in these 5 rows?

Reply:
Yes, prefer `局部` (not everywhere) to `可能` (not definitely) for местами everywhere, and I believe the same for `zh-tw`. @s8321414 any suggestions?

Reply:
hmmm, I don't think you should replace it everywhere :S

Reply:
@liyishuai @s8321414

Ok, I have changed it; now the only `Patchy` (`местами`) condition,
that does no use `局部` is this:

`392:小雷阵雪                               : Patchy light snow with thunder`

I don't know why. Is it correct, or should we add `局部` here too?

Reply:
In the Taiwan translation, they seem to use `部份` instead of `局部`.
Is it correct?

Reply:
392 should be `局部小雷阵雪`.

I know `部分` is also correct, but `部份` seems wrong, calling for native expert.

Reply:
Well, I updated some zh-tw translations in #421 :smiley:

Reply:
@liyishuai @s8321414 

I've merged the #421 pr, and updated it on the server.
Does it mean that we covered everything here,
or some mentioned problems are still open?

Reply:
Yes I think these are all addressed. Thanks!